static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -192,10 +192,8 @@ define([
                 if (config.page.contentType !== 'Network Front') {
                     history.logSummary(config.page);
                 }
-
-                if (config.page.contentType === 'Video') {
-                    history.logHistory(config.page);
-                }
+                
+                history.logHistory(config.page);
```

Comment:
Also, it seems from glancing at `history.js` (correct me if I'm wrong) that given that the max history size is 50, this will effectively make the video history much shorter (because the vast majority of the most recent 50 will be non-video)